packages/walletkit/package.json

```diff
@@ -9,10 +9,11 @@
   "scripts": {
     "build:for-env": "bin/build-sdk.js",
     "build:ts": "tsc -p .",
-    "clean": "rm -rf lib .artifacts contracts types lib",
-    "lint": "tslint -c tslint.json --project tsconfig.json",
     "prebuild": "yarn clean",
     "build": "yarn run build:for-env alfajores",
+    "clean": "rm -rf lib .artifacts contracts types lib",
+    "lint": "tslint -c tslint.json --project tsconfig.json",
+    "prepublishOnly": "yarn run prebuild && yarn run build",
```

Comment:
no need to do prebuild.
whenever you do `yarn run XXX`... all `preXXX` and `posXXX` are run automatically